README.rst

```diff
@@ -26,9 +26,12 @@ versions of the `LLVM-C API`_ should be basically effortless.
 
 To try it out install LLVM and get `llvmcpy` using `pip`::
 
-    sudo apt-get install llvm
+    sudo apt-get install llvm libllvm3.8
     pip install llvmcpy
 
+See [apt.llvm.org](http://apt.llvm.org/) for more package sources
```

Comment:
Beware: this is not Markdown, this is [reStructuredText](http://docutils.sourceforge.net/docs/user/rst/quickref.html).